### PR TITLE
add contributors logic / vis to storybook

### PR DESCRIPTION
**Pull request by jamesgorrie**
To allow a more visual understanding of the logic of contributors.
contributors

### CODE REVIEW

**Comment by jamesgorrie** [comment]
screen shot 2018-07-10 at 16 04 40
screen shot 2018-07-10 at 16 05 55

@GarethOrmerod @Heesoomoon ☝️

File: common/styles/utilities/_root-scope-classes.scss [hunk 1]
```diff
@@ -403,18 +403,20 @@
...
-  max-width: 100px;
+// Images
+.width-inherit {
+  width: inherit;
 }
 
-.half-in-image {
-  margin-top: -50px;
-}
+.avatar {
```
Comment (davidpmccormick) [inline common/styles/utilities/_root-scope-classes.scss]: I think I would sooner have this in it's own partial.

File: common/styles/utilities/_root-scope-classes.scss [hunk 2]
```diff
@@ -403,18 +403,20 @@
...
+  width: 78px;
+  height: 78px;
+  clip-path: inset($border-radius-unit round $border-radius-unit);
```
Comment (davidpmccormick) [inline common/styles/utilities/_root-scope-classes.scss]: Don't think this'll work in IE/Edge? Could you border-radius + overflow: hidden the wrapper instead?

File: common/styles/utilities/_root-scope-classes.scss [hunk 3]
```diff
@@ -403,18 +403,20 @@
...
+  transform: rotateZ(-6deg);
```
Comment (davidpmccormick) [inline common/styles/utilities/_root-scope-classes.scss]: is `rotateZ` better than `rotate`?

File: common/views/components/Contributor/Contributor.js [hunk 4]
```diff
@@ -12,53 +13,48 @@ const Contributor = ({
...
     alt: `Logo for ${contributor.name}`
   } : {
-    width: 64,
-    height: 64,
+    width: 78,
+    height: 78,
     contentUrl: contributor.image && contributor.image.contentUrl,
     alt: `Photograph of ${contributor.name}`
   };
+  // ${contributor.type === 'people' ?
```
Comment (davidpmccormick) [inline common/views/components/Contributor/Contributor.js]: ?

File: common/views/components/Contributor/Contributor.js [hunk 5]
```diff
@@ -12,53 +13,48 @@ const Contributor = ({
 }: ContributorType) => {
   const descriptionToRender = description || contributor.description;
   const imageProps: ImageProps = contributor.type === 'organisations' ? {
-    width: 64,
-    height: 64,
+    width: 78,
+    height: 78,
```
Comment (davidpmccormick) [inline common/views/components/Contributor/Contributor.js]: Do we want to be explicitly setting the image height? I'd say an incorrectly aspect-ratio-ed image would looke _more_ broken than a non-square image.

File: common/styles/utilities/_root-scope-classes.scss (hunk 1 above)
Comment (jamesgorrie) [reply common/styles/utilities/_root-scope-classes.scss]: > partial

❓

File: common/styles/utilities/_root-scope-classes.scss (hunk 2 above)
Comment (jamesgorrie) [reply common/styles/utilities/_root-scope-classes.scss]: Yea, I was going to say that IE doesn't get this magic...

We use clip-path on the images too - or do you think this integral / worth the effort to get it working in IE?

File: common/styles/utilities/_root-scope-classes.scss (hunk 3 above)
Comment (jamesgorrie) [reply common/styles/utilities/_root-scope-classes.scss]: Uses the GPU so smoother I think.

File: common/views/components/Contributor/Contributor.js (hunk 5 above)
Comment (jamesgorrie) [reply common/views/components/Contributor/Contributor.js]: I've left it on the contributor, as that's the decision taken by design.
I'll take a look at enforcing that we take a square crop.

File: common/views/components/Avatar/Avatar.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+type Props = {|
+  imageProps: ImageProps
+|}
+
+const style = {
+  width: sized(13),
+  height: sized(13),
+  clipPath: `inset(${sized(1)} round ${sized(1)})`,
+  transform: 'rotateZ(-6deg)'
```
Comment (davidpmccormick) [inline common/views/components/Avatar/Avatar.js]: sized?

**Pull request merged**